models/basic.py

```diff
@@ -65,23 +66,23 @@ class Basic(Model):
     """
 
 
-def create_model_for_test(props=None):
+def create_model(serial_obj=None, props=None, create_for_test=False,
```

Comment:
Also adding these params to remove the separate method for creating test models.
I think we can automate this so that we dont have to manually update each mode class but the problem is we need access to model constructor and it's args and their names defined in different files at run times.